src/ferm

```diff
@@ -2765,17 +2771,28 @@ sub execute_slow($$) {
     my $domain_cmd = $domain_info->{tools}{tables};
 
     if ($domain eq 'eb') {
-        my $tempfile = File::Temp->new(TEMPLATE => 'ferm.XXXXXXXXXX', TMPDIR => 1, OPEN => 0, UNLINK => 1);
-        my $filename = $tempfile->filename;
-        $domain_info->{ebt_current} = $tempfile;
-        $domain_cmd .= " --atomic-file $filename";
-        execute_command("$domain_cmd --atomic-init");
+        foreach my $eb_table (@eb_tables) {
+            my $tempfile = File::Temp->new(TEMPLATE => 'ferm.XXXXXXXXXX', TMPDIR => 1, OPEN => 0, UNLINK => 1);
+            my $filename = $tempfile->filename;
+            $domain_info->{ebt_current}->{$eb_table} = $tempfile;
+            execute_command("$domain_cmd -t $eb_table --atomic-file $filename --atomic-init");
+            execute_command("$domain_cmd -t $eb_table --atomic-file $filename --init-table");
+        }
     }
 
     my $status;
     while (my ($table, $table_info) = each %{$domain_info->{tables}}) {
         my $table_cmd = "$domain_cmd -t $table";
 
+        if ($domain eq 'eb') {
+            if ($table eq '') {
+                print "-->|$table|<--\n";
```

Comment:
What for?

Reply:
Oops, that is leftover from debugging. I updated the PR.